src/main/anovos/data_transformer/transformers.py

```diff
@@ -40,11 +40,6 @@
 from packaging import version
 from scipy import stats
 
-if version.parse(pyspark.__version__) < version.parse("3.0.0"):
-    from pyspark.ml.feature import OneHotEncoderEstimator as OneHotEncoder
-else:
-    from pyspark.ml.feature import OneHotEncoder
-
```

Comment:
As long as Anovos supports Spark 2.4.8, this switch has to remain in the code (See #356 for the plans to drop support for Python 3.7 and Spark 2.4.8)